fabio.properties

```diff
@@ -505,6 +506,46 @@
 # metrics.statsd.addr =
 
 
+# metrics.circonus.apiapp configures the API token app to use when
+# submitting metrics to Circonus. See: https://login.circonus.com/user/tokens
+# This is optional when ${metrics.target} is set to "circonus".
+#
+# The default is
+#
+# metrics.circonus.apiapp = "fabio"
```

Comment:
Don't use double quotes unless you want to send them. Also, if this is the default then you need to set this in `config/default.go`. Same for `metrics.circonus.apiurl`